docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 Sphinx==3.4.3
 sphinxcontrib-napoleon==0.7
 sphinx-rtd-theme==0.5.1
+autodoc==0.5.0
```

Comment:
Are you sure this is necessary?

I'm finding no mention of it in https://docs.readthedocs.io/en/stable/builds.html.... Also, the Autodoc project (https://pypi.org/project/autodoc/) seems to talk about documenting unit tests, which I'm pretty sure we don't want to do.

Reply:
Yup, you are right. I removed the autodoc dependency.